spec/lib/policy_actions/content_item_publisher_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+require 'gds_api/test_helpers/publishing_api'
+
+RSpec.describe ContentItemPublisher do
+  include GdsApi::TestHelpers::PublishingApi
+
+  let(:policy) { FactoryGirl.create(:policy) }
+  let(:content_item_publisher) { ContentItemPublisher.new(policy) }
+
+  describe 'run! method' do
```

Comment:
`"#run!"` is the usual way of writing this.  RSpec's printers know to deal with them specially.  Applies to the other policy actions.